DeepStorage/Patch_ListerMergeables_Notify_ThingStackChanged.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using HarmonyLib;
+using RimWorld;
+using Verse;
+
+namespace LWM.DeepStorage
+{
+    /// <summary>
+    /// Handles the splitoff and stack merge of spawned things on map.
+    /// </summary>
+    [StaticConstructorOnStartup]
+    public static class Patch_ListerMergeables_Notify_ThingStackChanged
+    {
+        static Patch_ListerMergeables_Notify_ThingStackChanged()
+        {
+            MethodInfo original = typeof(ListerMergeables).GetMethod(nameof(ListerMergeables.Notify_ThingStackChanged));
+            MethodInfo postfix = typeof(Patch_ListerMergeables_Notify_ThingStackChanged).GetMethod("Postfix", BindingFlags.Public | BindingFlags.Static);
+            Utils.HarmonyInstance.Patch(original, postfix: new HarmonyMethod(postfix));
```

Comment:
I would rather be uniform and use the [HarmonyPatch(...)] notation, but that's an easy fix.

Reply:
I can change them in the next pass.